modules/33acrossAnalyticsAdapter.js

```diff
@@ -300,6 +300,22 @@ function createReportFromCache(analyticsCache, completedAuctionId) {
     report.usPrivacy = uspDataHandler.getConsentData();
   }
 
+  const gdprConsentData = gdprDataHandler.getConsentData();
+  const gdprApplies = Boolean(gdprConsentData?.gdprApplies);
+  report.gdpr = Number(gdprApplies);
+  if (gdprApplies) {
+    report.gdprConsent = gdprConsentData.consentString || '';
+  }
+
+  if (gppDataHandler.getConsentData()) {
+    report.gpp = gppDataHandler.getConsentData().gpp;
+    report.gppSid = gppDataHandler.getConsentData().applicableSections.join(',');
```

Comment:
I think we are sending a POST request, isn't it? if that's the case you don't need to convert the applicableSections to a comma separated string, it's fine to leave them as an array.

Reply:
fixed